src/components/spreadsheet/utils/config-tables.js

```diff
@@ -1267,7 +1267,7 @@ export const TABLES_DEFINITIONS = {
                 getQuickFilterText: excludeFromGlobalFilter,
             },
             {
-                id: 'TargetV',
```

Comment:
Here you change the label for this column. You should not do it or validate this change with a PO